pkg/storage/bench_test.go

```diff
@@ -1697,7 +1703,12 @@ func runMVCCExportToSST(b *testing.B, opts mvccExportToSSTOpts) {
 		b.Fatal(err)
 	}
 
-	startWall := mkWall(opts.numRevisions/2 - 1) // exclusive, at least 1, so never see rangedels
+	// NB: startWall > 0 will enable the TBI optimization in
+	// MVCCIncrementalIterator. This has a significant penalty unless it can skip
+	// across large amounts of data. Here, TBI will be enabled for numRevisions>1,
+	// but not for numRevisions=1. Furthermore, numRevisions=1 will export MVCC
+	// range tombstones at the bottom, while numRevisions>1 will not.
```

Comment:
In my eyes, this makes it pretty difficult to reason across this benchmark when it really does very different things under the hood as the number of revisions goes up. If this is what it takes right now to get the benchmark comparable to 22.1 then so be it, though.

Reply:
Alternatively, we can have two versions of the benchmark: one which exports the entire dataset (disabling TBI), one which exports the top 50% (like it currently does).

Comparing with 22.1 still needs some result wrangling, and it would be possible to wrangle those results into the right form too.

Reply:
I'll defer to you in terms of comparing to the last release, but it would be good to have a clear benchmark going forward. Do we need to check in the exact code we're using to reason about the regression? Maybe we compare against last branch using code that's on a PR and check in code that actually gives a sane benchmark.
That said in the interest of time I'm fine merging this as is and going our merry ways if this is the code that allows for the comparison.

Reply:
Yeah, I'll add two variants: one for the entire dataset (no TBI) and one for the upper half (with TBI). But it's pretty far down on the list right now.

Reply:
Ok, updated with a percentage param, set to 100% (full export no TBI), 50% (incremental export with TBI, corresponds to 22.1), and 10%.